doc/source/examples/multiphysics/dam-break-VOF/dam-break-VOF.rst

```diff
@@ -295,7 +295,7 @@ the post-processing, with a very good agreement between the simulation and the e
 
```

Comment:
Above, I would specify the color of the phases (sorry I did not say that earlier): (do not commit that as is, put this above)

```suggestion
The following image shows the screenshots of the simulation at :math:`0`, :math:`1.1`, :math:`3`, and :math:`4` seconds, with the red area and the blue area corresponding conversely to the water location and the air location.
```